### PR TITLE
Add cdi-full group to passivation tests and move to full top package

### PR DESCRIPTION
**Pull request by starksm64**
Addresses #264, #273

Signed-off-by: Scott M Stark <starksm64@gmail.com>

### CODE REVIEW

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/Big.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating;
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/Big.java:17]: Double `passivating` is a mistake I guess?

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/PassivatingContextTest.java [hunk 2]
```diff
@@ -49,6 +50,7 @@
  * @author Martin Kouba
  */
 @SpecVersion(spec = "cdi", version = "2.0")
+@Test(groups = CDI_FULL)
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/PassivatingContextTest.java:53]: Sometimes, we add `(groups = CDI_FULL)` to an existing `@Test` annotation on a method, and sometimes we add a brand new `@Test(groups = CDI_FULL)` annotation on a class. Is there a difference? (Honest question, I don't really know :-) )

Comment (starksm64) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/PassivatingContextTest.java]: If there are multimple tests in a class I'm applying the group to, then I add it to the class level to avoid having to repeat it.

Comment (Ladicek) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/PassivatingContextTest.java]: Gotcha, thanks!

File: impl/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ejb-jar.xml [hunk 3]
```diff
@@ -8,8 +8,8 @@
 
       <session>
          <ejb-name>Elephant</ejb-name>
-         <business-local>org.jboss.cdi.tck.tests.context.passivating.enterprise.valid.ElephantLocal</business-local>
-         <ejb-class>org.jboss.cdi.tck.tests.context.passivating.enterprise.valid.Elephant</ejb-class>
+         <business-local>org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid.ElephantLocal</business-local>
+         <ejb-class>org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid.Elephant</ejb-class>
```
Comment (Ladicek) [inline impl/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ejb-jar.xml:12]: Still uses duplicate `passivating`.

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/Digital.java [hunk 4]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise;
```
Comment (Ladicek) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/Digital.java:17]: Here too.

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/DigitalInterceptor.java [hunk 5]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise;
```
Comment (Ladicek) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/DigitalInterceptor.java:17]: And here.

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/NonPassivationCapableSessionBeanTest.java [hunk 6]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise;
```
Comment (Ladicek) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/NonPassivationCapableSessionBeanTest.java:17]: And here.

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/Telephone.java [hunk 7]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise;
```
Comment (Ladicek) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/Telephone.java:17]: And here.

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/TelephoneLine.java [hunk 8]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise;
```
Comment (Ladicek) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/TelephoneLine.java:17]: And here.

File: web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ejb-jar.xml [hunk 9]
```diff
@@ -8,8 +8,8 @@
 
       <session>
          <ejb-name>Elephant</ejb-name>
-         <business-local>org.jboss.cdi.tck.tests.context.passivating.enterprise.valid.ElephantLocal</business-local>
-         <ejb-class>org.jboss.cdi.tck.tests.context.passivating.enterprise.valid.Elephant</ejb-class>
+         <business-local>org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid.ElephantLocal</business-local>
```
Comment (Ladicek) [inline web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ejb-jar.xml:11]: And here.

**Comment by Ladicek** [comment]
There's still a bunch of duplicate `passivating`, I marked most of them, but not all.

Some classes and `beans.xml` files appear moved, but that's just Git not being able to figure out what actually happened.

I also noticed some `beans.xml` files for EE-only tests exist both in `impl` and `web`, but that should probably be solved in another PR.

File: impl/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ejb-jar.xml (hunk 3 above)
Comment (starksm64) [reply impl/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ejb-jar.xml:12]: This needs to be moved into the web profile version of the tests as the core tests should have no dependencies on other containers.

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/decorator/enterprise/field/District.java [hunk 10]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.broken.interceptor.enterprise.field;
+package org.jboss.cdi.tck.tests.full.context.passivating.broken.decorator.enterprise.field;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/decorator/enterprise/field/District.java:17]: `interceptor` was updated to `decorator` by mistake, I think

**Comment by starksm64** [comment]
I'm closing this as the refactor that intellij did is wrong.

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/interceptor/enterprise/field/District.java [hunk 11]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.broken.decorator.enterprise.field;
+package org.jboss.cdi.tck.tests.full.context.passivating.broken.interceptor.enterprise.field;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/interceptor/enterprise/field/District.java:17]: `decorator` somehow became `interceptor`. Is this correct?

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/producer/field/enterprise/Cow.java [hunk 12]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.broken.producer.method.enterprise;
+package org.jboss.cdi.tck.tests.full.context.passivating.broken.producer.field.enterprise;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/producer/field/enterprise/Cow.java:17]: why changing `method` to `field`?

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/producer/method/enterprise/Cow.java [hunk 13]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.broken.producer.field.enterprise;
+package org.jboss.cdi.tck.tests.full.context.passivating.broken.producer.method.enterprise;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/broken/producer/method/enterprise/Cow.java:17]: Why did you change `field` to `method`?

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/Digital.java [hunk 14]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.valid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.invalid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/Digital.java:17]: double `passivating`

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/DigitalInterceptor.java [hunk 15]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.valid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.invalid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/DigitalInterceptor.java:17]: same here

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/Elephant.java [hunk 16]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.valid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.invalid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/Elephant.java:17]: same here

File: web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/decoratorWithNonPassivatingBeanConstructor/beans.xml [hunk 17]
```diff
@@ -2,6 +2,6 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" version="1.1" bean-discovery-mode="all">
    <decorators>
-      <class>org.jboss.cdi.tck.tests.context.passivating.broken.decoratorWithNonPassivatingInitializerMethod.CityDecorator</class>
+      <class>org.jboss.cdi.tck.tests.full.context.passivating.broken.decoratorWithNonPassivatingBeanConstructor.CityDecorator</class>
```
Comment (Emily-Jiang) [inline web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/decoratorWithNonPassivatingBeanConstructor/beans.xml:5]: `decoratorWithNonPassivatingBeanConstructor` -> `decoratorWithNonPassivatingInitializerMethod`

File: web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/decoratorWithNonPassivatingInitializerMethod/beans.xml [hunk 18]
```diff
@@ -2,6 +2,6 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" version="1.1" bean-discovery-mode="all">
    <decorators>
-      <class>org.jboss.cdi.tck.tests.context.passivating.broken.decoratorWithNonPassivatingBeanConstructor.CityDecorator</class>
+      <class>org.jboss.cdi.tck.tests.full.context.passivating.broken.decoratorWithNonPassivatingInitializerMethod.CityDecorator</class>
```
Comment (Emily-Jiang) [inline web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/decoratorWithNonPassivatingInitializerMethod/beans.xml:5]: same here

File: web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/enterpriseBeanWithNonPassivatingConstructorFieldInDecorator/beans.xml [hunk 19]
```diff
@@ -2,6 +2,6 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" version="1.1" bean-discovery-mode="all">
    <decorators>
-      <class>org.jboss.cdi.tck.tests.context.passivating.broken.enterpriseBeanWithNonPassivatingInitializerInDecorator.BrokenDecorator</class>
+      <class>org.jboss.cdi.tck.tests.full.context.passivating.broken.enterpriseBeanWithNonPassivatingConstructorFieldInDecorator.BrokenDecorator</class>
```
Comment (Emily-Jiang) [inline web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/enterpriseBeanWithNonPassivatingConstructorFieldInDecorator/beans.xml:5]: Why updating `enterpriseBeanWithNonPassivatingInitializerInDecorator` to `enterpriseBeanWithNonPassivatingConstructorFieldInDecorator`

File: web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/enterpriseBeanWithNonPassivatingInitializerInDecorator/beans.xml [hunk 20]
```diff
@@ -2,6 +2,6 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" version="1.1" bean-discovery-mode="all">
    <decorators>
-      <class>org.jboss.cdi.tck.tests.context.passivating.broken.enterpriseBeanWithNonPassivatingConstructorFieldInDecorator.BrokenDecorator</class>
+      <class>org.jboss.cdi.tck.tests.full.context.passivating.broken.enterpriseBeanWithNonPassivatingInitializerInDecorator.BrokenDecorator</class>
```
Comment (Emily-Jiang) [inline web/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/broken/enterpriseBeanWithNonPassivatingInitializerInDecorator/beans.xml:5]: same here

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/Telephone.java [hunk 21]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.valid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/Telephone.java:17]: double `passivating`

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/StatefulSessionBeanXmlDescriptorTest.java [hunk 22]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.valid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/StatefulSessionBeanXmlDescriptorTest.java:17]: double `passivating`

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/StatefulSessionBeanTest.java [hunk 23]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.valid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/StatefulSessionBeanTest.java:17]: double `passivating`

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ElephantLocal.java [hunk 24]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.valid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ElephantLocal.java:17]: double `passivating`

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/Elephant.java [hunk 25]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.invalid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/Elephant.java:17]: double `passivating`

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/DigitalInterceptor.java [hunk 26]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating.enterprise.invalid;
+package org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid;
```
Comment (Emily-Jiang) [inline web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/DigitalInterceptor.java:17]: double `passivating`

File: web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/Elephant.java (hunk 16 above)
Comment (Emily-Jiang) [reply web/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/invalid/Elephant.java:17]: many occurrences. please do a global replacement

**Comment by Ladicek** [comment]
I'd just like to point out that the change `decoratorWithNonPassivatingBeanConstructor` -> `decoratorWithNonPassivatingInitializerMethod` and many similar changes are, in fact, just an artifact of how Git handles file renames. It isn't tracked explicitly, and is instead inferred from file content changes. Sometimes, a mismatch like this happens.
